modules/logging-release-notes-5-8-0.adoc

```diff
@@ -56,6 +56,10 @@ In order to support multi-cluster log forwarding in additional namespaces other
 [id="logging-release-notes-5-8-0-known-issues"]
 == Known Issues
 
+* Currently, Splunk log forwarding might not work after upgrading to version 5.8 of the {clo}. This issue was caused by transitioning from OpenSSL version 1.1.1 to version 3.0.7. In the newer OpenSSL version, there is a default behavior change, where connections to TLS 1.2 endpoints are rejected if they do not expose the link:https://datatracker.ietf.org/doc/html/rfc5746[RFC 5746] extension.
```

Comment:
```suggestion
* Currently, Splunk log forwarding might not work after upgrading to version 5.8 of the {clo}. This issue is caused by transitioning from OpenSSL version 1.1.1 to version 3.0.7. In the newer OpenSSL version, there is a default behavior change, where connections to TLS 1.2 endpoints are rejected if they do not expose the link:https://datatracker.ietf.org/doc/html/rfc5746[RFC 5746] extension.
```
I would keep present tense here to avoid any confusion that the issue is resolved.